src/testcases/org/apache/poi/ss/usermodel/BaseTestSheetShiftColumns.java

```diff
@@ -0,0 +1,432 @@
+package org.apache.poi.ss.usermodel;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+
+import org.apache.poi.common.usermodel.HyperlinkType;
+import org.apache.poi.ss.ITestDataProvider;
+import org.apache.poi.ss.util.CellAddress;
+import org.apache.poi.ss.util.CellRangeAddress;
+import org.apache.poi.ss.util.CellUtil;
+import org.apache.poi.util.IOUtils;
+import org.apache.poi.xssf.XSSFITestDataProvider;
```

Comment:
this doesn't compile - the xssf classes are in poi-ooxml and the poi-ooxml depends on poi (but not vice versa)

Reply:
I was not aware of this. It's all one project and it compiles well on my machine.

Anyway, today I have fixed the issue, and separated xssf from ss. Now there are no xssf references in ss class BaseTestSheetShiftColumns.
Now there is new class TestXSSFSheetShiftColumns, and BaseTestSheetShiftColumns became abstract. 
TestXSSFSheetShiftColumns works fine, all tests pass. 
In TestHSSFSheetShiftColumns some tests fail, and that's because I did not implement everything about shifting for hssf scenario (and, as I told you before, I do not plan to implement it for now). So that class can be removed, or it can wait for someone to finish implementation.